examples/tests/youtube.test.ts

```diff
@@ -0,0 +1,26 @@
+// youtube test
+
+import { shortest } from "@antiwork/shortest";
+
+// Basic search and video playback
+shortest("Search for 'cute puppies' on YouTube and play the first video");
```

Comment:
Show casing how basic you can start with a test case.

Reply:
> Show casing how basic you can start with a test case.

Doesn't read like English. What do you mean?